displayio_cartesian.py

```diff
@@ -111,56 +123,6 @@ class Cartesian(Widget):
 
         display.show(my_group) # add the group to the display
 
-
-    **Summary: Cartesian Features and input variables**
-
-    The `Cartesian` widget has some options for controlling its position, visible appearance,
-    and scale through a collection of input variables:
-
-        - **position**: ``x``, ``y``, ``anchor_point``, ``anchored_position`` and
-          ``nudge_x``, ``nudge_y``. Nudge parameters are used to account for the float and int
-          conversions required to display different ranges and values. Conversion are required
-          as displays work in integers and not floats
-
-        - **size**: ``width`` and ``height``
-
-        - **color**: ``axes_color``, ``font_color``, ``tick_color``, ``pointer_color``
-
-        - **background color**: ``background_color``
-
-        - **linewidths**: ``axes_stroke`` and ``major_tick_stroke``
-
-        - **range**: ``xrange`` and ``yrange`` This is the range in absolute units.
-          For example, when using (20-90), the X axis will start at 20 finishing at 90.
-          However the height of the graph is given by the height parameter. The scale
-          is handled internal to provide a 1:1 experience when you update the graph.
-
-
-    .. figure:: cartesian.gif
-       :scale: 100 %
-       :figwidth: 50%
-       :align: center
-       :alt: Diagram of the cartesian widget with the pointer in motion.
-
-       This is a diagram of a cartesian widget with the pointer moving in the
-       plot area.
-
-    .. figure:: cartesian_zones.png
-       :scale: 100 %
-       :figwidth: 50%
-       :align: center
-       :alt: Diagram of the cartesian widget zones.
-
-       This is a diagram of a cartesian widget showing the different zones.
-
-    .. figure:: cartesian_explanation.png
-       :scale: 100 %
-       :figwidth: 50%
-       :align: center
-       :alt: Diagram of the cartesian widget localisation.
-
-       This is a diagram of a cartesian widget showing localisation scheme.
-
```

Comment:
Do we want all of this removed from the class file for a specific reason? I think it's kind of convenient to have the documentation contained in the code file so that we can see the docs along with the implementation without having to click around to different places.

Reply:
Probably right it is easier, even tough. The only reason is to be closer to all our other libraries. also this is a personal preference, I love the way the GPS documentation is done, and this is just a humble copy of that model.
Another thing also,  In our library design Guidelines the concept of documentation was not present before, and at the moment we only have the section in how to use the library. https://github.com/adafruit/circuitpython/blob/main/docs/design_guide.rst.
So just want to be consistent across the libraries